_literate/A-model-tuning/tutorial.jl

```diff
@@ -86,18 +86,12 @@ r = report(m)
 r.best_history_entry.measurement[1]
 
 # Anyone wants plots? of course:
-
-using PyPlot
-ioff() # hide
-figure(figsize=(8,6))
+using Plots
 res = r.plotting # contains all you need for plotting
-plot(res.parameter_values, res.measurements, ls="none", marker="o")
-
-xticks(1:5, fontsize=12)
-yticks(fontsize=12)
-xlabel("Maximum depth", fontsize=14)
-ylabel("Misclassification rate", fontsize=14)
-ylim([0, 1])
+plot(vec(res.parameter_values), res.measurements, seriestype=:scatter, markershape=:circle, ylim=(0,1),
```

Comment:
How about we just use `plot(m)` here?

Reply:
Sure. I did that because I was just reproducing the existing plot.